### DISCUSSION TITLE
save files locally [Jun 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2433] [Discord Threads]
> 🙋‍♂️ **peter piper**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@820992975298625547> could you please elaborate on requirement & need? help us understand   
#### by files: which files are we referring here? nocodb database file? or attachments associated with nocodb project?  
#  
> 💁 **o1lab**  
#### If it's attachment .. S3 or similar is not installed.. it will store on the filesystem of your server  
#  
> 💁 **o1lab**  
#### See our docs  
#  
> 💁 **o1lab**  
#### On how to mount that folder  
#  
> 💁 **o1lab**  
#### In installation  
#  
> 🙋‍♂️ **peter piper**  
#### <@857998597474484295> export all the data in your database to a sql file and then tar it up. this can be done from a command, which you can add to a crontab  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984901501644341358)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
